skimage/segmentation/random_walker_segmentation.py

```diff
@@ -290,21 +319,25 @@ def random_walker(data, labels, beta=130, mode='bf', tol=1.e-3, copy=True,
         If True, input data is parsed as multichannel data (see 'data' above
         for proper input format in this case).
     return_full_prob : bool, optional
-        If True, the probability that a pixel belongs to each of the labels
-        will be returned, instead of only the most likely label.
+        If True, the probability that a pixel belongs to each of the
+        labels will be returned, instead of only the most likely
+        label. The returned probability values may be inaccurate
```

Comment:
hum, this sentence is worrying but not very informative... I think we should raise a warning instead as was suggested in #2360.

Reply:
I agree, but we have to set a tolerance above which the warning is raised, because it is not rare that the returned "probability" value is > 1!
The problem is in the use of the word "probability" since the method doesn't give any guaranty that the result is in the range (0, 1).
I see in the [ref](http://vision.cse.psu.edu/people/chenpingY/paper/grady2006random.pdf) page 6 that the author uses the term "probabilities/potentials". Can't we use this wording instead?

Reply:
How about we just test whether the returned probability is >1 (possibly with some small tolerance like 1.e-4)? It does not have to be in this PR, it's just that it'd prefer to have a warning rather than a sentence which will mostly scare people or be unnoticed.

Reply:
I agree that the sentence that I introduced can be discouraging...
I am not against raising a warning, I am just saying that if we replace the argument name`return_full_prob` by ` return_full_potentials`, we alleviate the need for ensuring that the returned values are in the range (0, 1)...
May be someone else vote can ease our decision :-)

Reply:
I suggest we remove this comment for now (it is a separate issue from this PR), and create a PR specifically for it. `return_full_potentials` is not a bad choice, but it's unclear that it's worth a deprecation cycle. It's good to keep PRs as small and self-contained as possible!